tests/end2end/playwright/pages/selectionpage.js

```diff
@@ -0,0 +1,131 @@
+// @ts-check
+import { ProjectPage } from './project';
+
+/**
+ * Playwright Page
+ * @typedef {import('@playwright/test').Page} Page
+ */
+
+/**
+ * Playwright Page
+ * @typedef {import('@playwright/test').Locator} Locator
+ */
+
+export class SelectionPage extends ProjectPage {
+    // Metadata
+    /**
+     * The print panel
+     * @type {Locator}
+     */
+    selectionPanel;
+
+    /**
+     * The print button on menu
+     * @type {Locator}
+     */
+    selectionSwitcherButton;
+
+    /**
+     * Constructor for a QGIS project page
+     * @param {Page} page The playwright page
+     * @param {string} project The project name
+     * @param {string} repository The repository name, default to testsrepository
+     */
+    constructor(page, project, repository = 'testsrepository') {
+        super(page, project, repository);
+
+        this.selectionPanel = page.locator('#selectiontool');
+        this.selectionSwitcherButton = page.locator('#button-selectiontool');
+    }
+
+    /**
+     * Returns the selection panel
+     * @returns {Promise<Locator>} The selection panel locator
+     * @example
+     * const selectionPanel = await project.getSelectionPanel();
+     * await expect(selectionPanel).toBeVisible();
+     */
+    async getSelectionPanel() {
```

Comment:
I'm not fan of this prefix naming `get`. A lot of projects do not use this kind of prefix, like QGIS for instance.
We should decide, _later_, to keep consistency across the project.

Just for information

Reply:
Done

Reply:
I was meaning for all other `get` methods you added, not just this one.